src/stream/src/executor/managed_state/aggregation/mod.rs

```diff
@@ -143,19 +143,19 @@ pub enum ManagedStateImpl<S: StateStore> {
 }
 
 impl<S: StateStore> ManagedStateImpl<S> {
-    pub async fn apply_batch(
+    pub async fn apply_chunk(
         &mut self,
         ops: Ops<'_>,
         visibility: Option<&Bitmap>,
-        data: &[&ArrayImpl],
+        columns: &[&ArrayImpl],
         epoch: u64,
```

Comment:
Looks like the `epoch` is never used now? 🤔

Reply:
yep, it never reads from state table in `apply_chunk`